test/cqlpy/cassandra_tests/vector_invalid_query_test.py

```diff
@@ -0,0 +1,548 @@
+# This file was translated from the original Java test from the Apache
+# Cassandra source repository, as of commit bb66561142788270ab450c02de836b3952ed37b4
+#
+# The original Apache Cassandra license:
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from .porting import *
+
+# public class VectorInvalidQueryTest extends SAITester
+# {
+#     @BeforeClass
+#     public static void setupClass()
+#     {
+#         requireNetwork();
+#     }
+
+#     @Test
+#     public void cannotCreateEmptyVectorColumn()
+#     {
+#         assertThatThrownBy(() -> execute(String.format("CREATE TABLE %s.%s (pk int, str_val text, val vector<float, 0>, PRIMARY KEY(pk))",
+#                                                        KEYSPACE, createTableName())))
+#         .isInstanceOf(InvalidRequestException.class)
+#         .hasMessage("vectors may only have positive dimensions; given 0");
+#     }
```

Comment:
It's nice that you copied over the original tests and included them, in a commented form, in this commit - this helps review the code that you rewrote. However, I believe that ultimately we shouldn't keep the commented-out java tests - we might want to adjust some of them at some point, and the original ones are still available in the Cassandra repository.

My suggestion is as follows:

- Introduce a commit which only contains original tests in a commented form; the only non-comment line being `from .porting import *`,
- Replace some of the commented out tests with their python equivalents.

This way reviewers can validate the translation easily and we don't get duplicated commented code at the end.